backend/src/models/item/postgres.js

```diff
@@ -0,0 +1,85 @@
+// Wrapper around postgres data functions for the Item Model. Requires a
+// postgres connection and ability to make valid postgres queries.
+
+const ItemRepo = (postgres) => {
+  //Setup the item table if it does not exist
+  const createItemTableSQL = `
+    CREATE TABLE IF NOT EXISTS items(
+      item_id SERIAL PRIMARY KEY,
+      item_name text NOT NULL,
+      seller_id integer NOT NULL,
+      pic_url text,
+      item_description text,
+      tags text,
+      sale_price float NOT NULL,
+      ticket_price float,
+      total_tickets integer NOT NULL,
+      bids text,
```

Comment:
Bids is going to have to be an array of integers, because they are IDs of the bids table which we'll implement later. Not sure how to do that in Postgres though